screens/SettingsScreen/SettingsScreen.tsx

```diff
@@ -0,0 +1,697 @@
+            </Headline>
+          </View>
+          <View style={styles.flexRow}>
+            <Caption>{"-insert email here-"}</Caption>
+          </View>
+        </View>
+      </View>
+    )
+  }
+
+  const ChangePasswdSection = () => {
+    return (
+      <View
+        style={[
+          styles.flexRow,
+          styles.paddingIconSection,
+          { alignItems: "center", justifyContent: "center" }
+        ]}
+      >
+        <View style={[styles.spacing, { paddingHorizontal: 10 }]}>
+          <View style={{ justifyContent: "center" }}>
+            <Ionicon
+              style={[styles.iconStyling]}
+              name="shield"
+              color={"black"}
+            />
+          </View>
+        </View>
+
+        <View style={[styles.spacing, { paddingHorizontal: 2 }]}>
+          <View style={{ marginBottom: 4 }}>
+            <Headline
+              style={[
+                styles.textColor,
+                { textAlignVertical: "center", textAlign: "center" }
+              ]}
+            >
+              {"Change Password"}
+            </Headline>
+          </View>
+        </View>
+
+        <View
+          style={[
+            styles.flexRow,
+            { flex: 1, marginRight: 8, justifyContent: "flex-end" }
+          ]}
+        >
+          <Ionicon
+            name="chevron-forward"
+            style={{ alignSelf: "center" }}
+            color={AppStyles.colorOpacity35}
+          />
+        </View>
+      </View>
+    )
+  }
+
+  const BlockedUserSection = () => {
+    return (
+      <View
+        style={[
+          styles.flexRow,
+          styles.paddingIconSection,
+          { alignItems: "center", justifyContent: "center" }
+        ]}
+      >
+        <View
+          style={[
+            styles.spacing,
+            { paddingVertical: 0, paddingHorizontal: 10 }
+          ]}
+        >
+          <View style={{ justifyContent: "center" }}>
+            <Ionicon
+              style={[styles.iconStyling]}
+              name="person-remove"
+              color={"black"}
+            />
+          </View>
+        </View>
+
+        <View style={[styles.spacing, { paddingHorizontal: 2 }]}>
+          <View style={{ marginBottom: 0 }}>
+            <Headline
+              style={[
+                styles.textColor,
+                { textAlignVertical: "center", textAlign: "left" }
+              ]}
+            >
+              {"Blocked Users"}
+            </Headline>
+          </View>
+        </View>
+
+        <View
+          style={[
+            styles.flexRow,
+            { flex: 1, marginRight: 8, justifyContent: "flex-end" }
+          ]}
+        >
+          <Ionicon
+            name="chevron-forward"
+            style={{ alignSelf: "center" }}
+            color={AppStyles.colorOpacity35}
+          />
+        </View>
+      </View>
+    )
+  }
+
+  const EventNotificationSection = () => {
+    const [eventScreenNotifEnabled, seteventScreenNotifEnabled] =
```

Comment:
Use atomWithStorage (make sure to place these outside of the components themselves)